travvy-server/src/controllers/AuthenticationController.js

```diff
@@ -46,13 +49,14 @@ module.exports = {
         return user
     },
 
-    async onboarding (email, age, gender, instagram_username, preferred_activity_type) {
-        //here we find a user who's email matches an email in the user database
+    async onboarding (email, age, gender, instagram_username, preferred_activity_type, activity_id) {
+     
         const user = await Users.update({
             Age: age,
             Gender: gender,
             Instagram_Username: instagram_username,
             Preferred_Activity_Type: preferred_activity_type,
+            activity_id: activity_id,
```

Comment:
This is great so we can update it with the onboarding page! We should also look into letting users see and update it on the profile page

Reply:
Yes great idea, that could be implemented in the future!

Reply:
Daria implemented this! It was one of her tickets this sprint :)